lib/ace/mode/csp_highlight_rules.js

```diff
@@ -0,0 +1,214 @@
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /disown-opener/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/disown-opener"
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /form-action/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/form-action"
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /frame-ancestors/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors"
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /report-uri/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri"
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /report-to/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-to"
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /upgrade-insecure-requests/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/upgrade-insecure-requests"
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /block-all-mixed-content/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/block-all-mixed-content"
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /require-sri-for/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/require-sri-for"
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /reflected-xss/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/reflected-xss"
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /referrer/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/"
+                },
+                {
+                    token: "constant.language.character",
+                    regex: /policy-uri/,
+                    comment: "https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/policy-uri"
```

Comment:
would be better to remove comment properties and just keep one url in a comment, instead of the comment on line 31

Reply:
This is a bit unclear, you say to remove the `comment` property. That I get, I don't get what should I do with the URL? Because on line 31 o this file I have `/* This file was autogenerated from  (uuid: ) */`. Should I just remove the comment section all-together?

Reply:
You can remove the `/* This file was autogenerated from (uuid: ) */` comment and the one after it. And instead of them add `// see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/` comment.

Reply:
I see. I thought that since some parts of the file are auto generated I should not touch those parts.